### DISCUSSION TITLE
Personal website 404 error when repo is set to private

### DISCUSSION THREAD

**Discussion by leee222** [community/community#36070] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

My github `index.html` can be correctly built for my personal website when the repository is set public. But when I change the repository to private the website returns 404 error. Is it supposed to work this way? if so that means anyone can edit our personal website so I believe there might be an issue under my account.

Another thing to note is my account used to be under github education. At that time I didn't have this problem - I could publish my personal website from private repository. 

But now my account isn't under github education and the personal website returns 404 error with the private repository.

Any help would be welcome. Thanks

**Comment by antonblr** [comment, marked as answer]
Hi @leee222 - that is correct behavior. Per [About GitHub Pages](https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/about-github-pages) : 

>GitHub Pages is available in **public** repositories with GitHub Free and GitHub Free for organizations, and in **public and private** repositories with **GitHub Pro, GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server**. For more information, see "[GitHub's products](https://docs.github.com/en/enterprise-cloud@latest/get-started/learning-about-github/githubs-products)."

And no, no one will be able edit your public website if you create [branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches), require Pull Requests, require codeowners approvals etc

**Reply by leee222** [reply]
Hi @antonblr 
Didn't know about branch protection rules before. Thanks for the answer!
